### ISSUE TITLE
Crease blender to zbrush it's nor working! T_T.

### ISSUE THREAD

**Issue by SbstnDC** [JoseConseco/GoB#107]
Hello, it happens to me that when I export a mesh with crease, zbrush does not recognize it, I am with 2020.4, and blender 2.90a.

**Comment by kromar** [comment]
@SbstnDC i dont think there is much that i can do abut this, the GoZ format has no edge information so i doubt that its possible to send crease information. Or did this work in older versions?

**Comment by SbstnDC** [comment]
@kromar 
ohh it is true, I have tried previous versions and it has not worked, I think this time I export without applying modifiers, so I thought that crease was exported, my mistake.

 Anyway, it would be interesting if it is possible to send the crease to zbrush to avoid applying it manually.

 Thank you

**Comment by kromar** [comment]
Yes it would be indeed and i will have a look if it could be somehow possible. However the limitation here will be the GoZ format and what zbrush does with it so not much we can do within that.
However it might be possible to run it as a zscript after the export has been done. Since i have 0 experience with that part i cant tell whats possible at this point.

**Comment by Tsukasa-YAMATO** [comment]
@SbstnDC 
In general, addon’s developers do not support "alpha version" because of many many troublesome for tracing and dealing with daily changing of alpha versions.

And also in general as well, 3DCG software’s crease settings are almost all unique to the software itself and are often incompatible. I recognize the same with Blender and Zbrush.

For example, is it support crease settings in OBJ format or FBX?( I don’t know well. They are not supported I think.) Crease settings compatibility may be quite low.

I don't think there will be a day when it can deal with it, so I recommend that you try to find a solution by yourself. :)
